code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -202,6 +202,31 @@
 	else
 		..()
 
+///Wisdom cow, gives XP to a random skill and speaks wisdoms
+/mob/living/simple_animal/cow/wisdom
+	name = "wisdom cow"
+	desc = "Known for its wisdom, shares it with all"
+	gold_core_spawnable = FALSE
+	tame_chance = 0
+	bonus_tame_chance = 0
+	speak_chance = 15
+
+/mob/living/simple_animal/cow/wisdom/Initialize()
+	. = ..()
+	speak = GLOB.wisdoms //Done here so it's setup properly
+
+///Give intense wisdom to the attacker if they're being friendly about it
+/mob/living/simple_animal/cow/wisdom/attack_hand(mob/living/carbon/M)
+	if(!stat && M.a_intent == INTENT_HELP)
+		to_chat(M, "<span class='nicegreen'>[src] whispers you some intense wisdoms and then dissapears!</span>")
+		M.mind?.adjust_experience(pick(subtypesof(/datum/skill)), 500)

```

Comment:
The problem is there's no shared scale of experience between different skills iirc, you may be better off increasing their experience in a skill based on a percentage of their current skill. Increase their exp by 10% perhaps